apps/alerts/lib/cache/bus_stop_change_s3.ex

```diff
@@ -0,0 +1,128 @@
+defmodule Alerts.Cache.BusStopChangeS3 do
+  @moduledoc """
+  Functionality for writing to/reading from S3. Takes %Alert{} converts them
+  into %HistoricalAlert{} before saving.
+  """
+  use RepoCache, ttl: :timer.hours(24)
+  require Logger
+  alias Alerts.{Alert, HistoricalAlert}
+
+  @ex_aws Application.compile_env(:alerts, [:mock_aws_client], ExAws)
+  @ex_aws_s3 Application.compile_env(:alerts, [:mock_aws_client], ExAws.S3)
+  @bucket "mbta-dotcom"
+  @bucket_prefix Application.compile_env!(:alerts, [:bus_stop_change_bucket])
```

Comment:
@thecristen I'm surprised that this didn't generate a warning on startup, but you're getting the change prefix at compile time. That means you can't override it with the environment variable at runtime.